api/backend_test.go

```diff
@@ -5,11 +5,23 @@ import (
 	"encoding/json"
 	"net/http"
 	"net/http/httptest"
+	"os"
+	"os/exec"
 	"reflect"
 	"strings"
 	"testing"
 )
 
+func TestMain(m *testing.M) {
+	cmd := exec.Command("python3", "get_binaries.py")
+	err := cmd.Run()
+	if err != nil {
+		os.Exit(1)
+	}
+	os.Exit(m.Run())
```

Comment:
Here, wouldn't we run `m.Fatal(err)` to display the err to the tester?

Reply:
The type *testing.M does not have that method.